receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -62,11 +64,18 @@ func newMetricFamily(metricName string, mc MetadataCache) MetricFamily {
 			metadata.Metric = familyName
 			metadata.Type = textparse.MetricTypeUnknown
 		}
+	} else if !ok && isInternalMetric(metricName) {
+		metadata = defineInternalMetric(metricName, metadata, logger)
+	}
+	//TODO convert it to OtelMetrics ?
```

Comment:
If I understand this comment correctly, this is tracked here: https://github.com/open-telemetry/opentelemetry-collector/issues/3137.    If that is what you are trying to capture, we can remove this TODO, and use the issue to track.